src/js/components/DataChart/doc.js

```diff
@@ -87,8 +178,31 @@ export const doc = DataChart => {
       ]),
       PropTypes.string,
     ]).description(`The spacing between the axes and the Charts.`),
+    guide: PropTypes.oneOfType([
+      PropTypes.bool,
+      PropTypes.shape({
+        x: PropTypes.oneOfType([
+          PropTypes.bool,
+          PropTypes.shape({
+            granularity: granularityType,
+          }),
+        ]),
+        y: PropTypes.oneOfType([
+          PropTypes.bool,
+          PropTypes.shape({
+            granularity: granularityType,
+          }),
+        ]),
+      }),
+    ]).description(`Whether to put guidelines underneath the chart graphics.
+    See the description of 'granularity' under 'axis'.`),
+    legend: PropTypes.bool.description('Whether to include a legend'),
     pad: padPropType.description(`Spacing around the outer edge of
     the drawing coordinate area for the graphic elements to overflow into.`),
+    series: PropTypes.oneOfType([
+      seriesType,
+      PropTypes.arrayOf(seriesType),
+    ]).description('TBD'),
```

Comment:
Can we fill in the TBD for `series` here?

Reply:
fixed